README.md

```diff
@@ -144,5 +161,10 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
 ```js
-// Your answer goes here...
+  $("psot").click
+    $.ajax({
+      type: "POST",
+      url: "/index",
+      data: { product: { title: "someTitle", body: "someBody" } },
+  })
```

Comment:
Your making the AJAX call correctly, but what about printing the new post to the browser console?